src/components/containers/Panel/Panel.module.scss

```diff
@@ -11,7 +11,7 @@
   overflow: hidden;
   
   min-height: 4rem;
-  padding: 22px 26px; // FIXME
+  padding: bk.$spacing-4;
```

Comment:
Shouldn't the padding here match the `Card` one?

Reply:
It's following ticket & Figma :)